lib/view/repositories_screen.dart

```diff
@@ -1,15 +1,19 @@
 import 'package:flutter/material.dart';
 
-import '../widgets/drop_down.dart';
+import '../view_model/repositories_view_model.dart';
+import 'widget/drop_down.dart';
+import 'widget/repositories_widget.dart';
 
 class RepositoriesScreen extends StatefulWidget {
   const RepositoriesScreen({super.key});
+  static const routeName = "repositories_screen";
 
   @override
   State<RepositoriesScreen> createState() => _RepositoriesScreenState();
 }
 
 class _RepositoriesScreenState extends State<RepositoriesScreen> {
+  final RepositoriesViewModel viewModel = RepositoriesViewModel();
```

Comment:
As my other suggestion, I would recommend you to receive the viewModel by parameter. In that case, you should move the view model to the widget, instead of the state.

From the state, you can access the widget information calling `widget.viewModel`.

Reply:
done